pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
```

Comment:
Is this the version that first included distutils?  How did you pick the version?

Reply:
I copied this from https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#declaring-the-build-backend.